modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -34,78 +34,80 @@ $ oc describe deployment gitops-operator-controller-manager -n openshift-operato
+* Before this update, there was no option for choosing an algorthm for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to `round-robin`, which distributes clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
 
-* With this update, the {gitops-title} Operator will exports a few custom metrics that will allow you to track the performance of the operator. These metrics include:
-** `active_argocd_instances_total` - The number of Argo CD instances currently being managed across the cluster.
-** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase.
-** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace has been reconciled.
-** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their durations for the instance in a given namespace. link:https://issues.redhat.com/browse/GITOPS-3071[GITOPS-3071]
-
-* With this update, the Argo CD CRD API version has been upgraded from v1alpha1 to v1beta1 in order to accomodate the breaking changes resulting from the deprecation of `.spec.dex` and certain `.spec.sso` fields. To streamline the automatic migration of existing v1alpha1 ArgoCD CRs to v1beta1, conversion webhook support has been implemented. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
-+
-[NOTE]
-====
-By default, the conversion webhook is enabled only for OLM-installed operators. For non-OLM installations of the operator, enabling the webhook is optional. However, without conversion webhook support, you are responsible for manually migrating any existing ArgoCD v1alpha1 CRs to v1beta1.
-====
+* With this update, the {gitops-title} Operator exports custom metrics that allow you to track the performance of the Operator. These metrics include:
+** `active_argocd_instances_total` - The number of Argo CD instances currently managed across the cluster
+** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase
+** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace is reconciled
+** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their duration for the instance in a given namespace
```

Comment:
Punctuate the sentences with a period. I see that in history you have the punctuation. 

@ekristova Any reason why the punctuation is removed? I see that they read as complete sentence. Am I missing somethign here?